plugin/storage/es/esRollover.py

```diff
@@ -69,6 +69,10 @@ def main():
 
 def perform_action(action, client, write_alias, read_alias, index_to_rollover, template_name):
     if action == 'init':
+        if not is_alias_empty(client, read_alias) or not is_alias_empty(client, write_alias):
+            print("Alias {} or {} is not empty aborting init step".format(read_alias, write_alias))
```

Comment:
Shouldn't instead the creation of the index be made conditional? If this script fails after creating a read alias but before write alias there's no way to fix the situation by rerunning it.

Reply:
+1. I think that was the initial goal. I have reverted this and fixed catching the exception in the create index func